report.php

```diff
@@ -134,12 +139,36 @@
         }
         $row[] = logstore_xapi_get_info_string($result);
         $row[] = userdate($result->timecreated);
+
+        $row[] = '<span id="reply-event-id-' . $result->id . '" class="reply-event"></span>';
         $table->data[] = $row;
     }
-    echo html_writer::start_tag('div', array('class'=>'no-overflow'));
+}
+
+// Create page view.
+$PAGE->set_context($systemcontext);
+$PAGE->set_url($baseurl);
+$PAGE->set_pagelayout('report');
+
+$PAGE->requires->js_call_amd('logstore_xapi/replayevents', 'init', [$eventids]);
```

Comment:
Issue here:
On the form I need a sum and perhaps all id of events but I don't have an idea how I can get it on the form easily.
thats why I pushed it in the js call. But it is too slow. Any better idea should be good for me.

When the form is created we don't know the selectors how are set.

Reply:
Could you use data attributes or perhaps hidden fields, depending on what you are needing to do with this?

Reply:
Because of the 'id updating' (when event processing run) I don't need this element at all.
I will remove it in the next PR. because it is in connect with the multiple replaying.

Reply:
I am not sure how I can use data attributes in this case.
Could you give an explain?

Reply:
https://developer.mozilla.org/en-US/docs/Learn/HTML/Howto/Use_data_attributes

Something like this is how I used to use PHP to generate data and store it on the page for reference by JS

Reply:
I see what you mean now.